desc/grid.py

```diff
@@ -1123,7 +1161,9 @@ def _create_nodes(  # noqa: C901
                 dt *= t.size / (t.size - 1)
                 # scale_weights() will reduce endpoint (dt[0] and dt[-1])
                 # duplicate node weight
-        else:
+            # if custom theta used usually safe to assume its non-uniform so no fft
+            self._fft_poloidal = (not endpoint) and (not self.sym)
```

Comment:
Won't this still evaluate to `fft_poloidal = True` even if the nodes are non-uniform?